src/index.js

```diff
@@ -0,0 +1,4 @@
+module.exports = function(config) {
+  "use strict";
```

Comment:
What's this file doing?

Reply:
My mistake. Here need to export commandHandler.